ynh_add_secure_repos/ynh_add_secure_repos

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+ 
+ynh_add_repos(){
+  name_repos="$1"
+  url_repos="$2"
+  echo "# $name_repos
+    ${url_repos}" 
+    > "/etc/apt/sources.list.d/$name_repos.list"
+}
+
+ynh_add_pin_repo() {
+  name_repos="$1"
+  pin="$2"
+  origin="$3"
+  echo "Package: *
+       Pin: release o=$origin,a=$name_repos
+       Pin-Priority: $pin" \
+    > "/etc/apt/preferences.d/$name_repos"
+}
+
+ynh_add_secure_backport() {
+   name_repos="debian-backports"
+   origin="Debian"
+   lsb_version="$(ynh_get_debian_release)"
+   url="deb https://ftp.debian.org/debian $lsb_version-backports main"
```

Comment:
About the URL `https://ftp.debian.org/debian` don't know if it's a good Idea to use a URL written in hard. I purpose to use some thing like that : 
1. Try to get the URL from the debian stable repos.
2. If we find, it use it, when else use mybe this.

Reply:
We can try to search the mirror_director url perhaps ?

Reply:
Ok, on my side. It's just that I don't really like a URL how is hard written in the code.

Reply:
Yes, but it's specific to backports.... for other repos you can use ynh_add_repos directly + ynh_add_pin_repo.